src/MAVN.Service.SmartVouchers.DomainServices/VouchersService.cs

```diff
@@ -17,9 +17,11 @@
 
 namespace MAVN.Service.SmartVouchers.DomainServices
 {
-    public class VouchersService : IVouchersService
+    public class   VouchersService : IVouchersService
     {
         private const int MaxAttemptsCount = 5;
+        private const string SuccessPaymentStatus = "success";
```

Comment:
It's "Success" and "Pending"

Reply:
I am comparing it with the received status .ToLower() that's why they are with lower case